pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -179,7 +179,7 @@ Each evaluation step, then, transforms the current set of definitions and
 program into a new set of definitions and program. Before a
 @racket[define] can be moved into the set of definitions, its
 expression (i.e., its right-hand side) must be reduced to a @tech{value}.
-(The left-hand side is not evaluated.)
+(The left-hand side is not an expression position, and so it is not evaluated.)
```

Comment:
Trying to address @lexi-lambda's comment, since I felt similarly about the change.